solo_study.md

```diff
@@ -48,16 +48,16 @@ Today you objectives are to put in practice the concepts you learned yesterday.
 
 *Note: Refer to this handy [Javascript Cheat sheet](http://overapi.com/javascript) if you need it*
 
-### Plan
+### Plan  http://jsbin.com/vetazuv/6/edit?js,console
 
-- [ ] Complete the last 5 tasks of [Objects Basics Tutorial](https://learn.javascript.info/object)
-  - [ ] Link to your solution here (jsbin, gist, or a js file in this repo)
-- [ ] Complete the last 5 tasks of [Arrays Tutorial](https://learn.javascript.info/array)
- - [ ] Link to your solution here
-- [ ] Complete the last 5 tasks of [Array Methods](https://learn.javascript.info/array-methods)
- - [ ] Link to your solution here
-- [ ] Complete the [Learn JS Basics tutorial](http://www.learn-js.org/en/Welcome)
-- [ ] Complete the [Learn JS Objects tutorial](http://www.learn-js.org/en/Object_Oriented_JavaScript)
+- [X] Complete the last 5 tasks of [Objects Basics Tutorial](https://learn.javascript.info/object)
+- [X ]   solution http://jsbin.com/vetazuv/6/edit?js,console or https://repl.it/GIOI/0
+- [X] Complete the last 5 tasks of [Arrays Tutorial](https://learn.javascript.info/array)
+- [X] solution https://repl.it/GIOP/6
+- [X] Complete the last 5 tasks of [Array Methods](https://learn.javascript.info/array-methods)
+ - [ ] solution herehttps://repl.it/GIQN/2
```

Comment:
The camelcase function isn't putting the words into camelCase (lowercase first word, uppercase second word and onward)

I'd recommend doing a for loop that ignores the first word in the array, so it starts at index 1 instead of 0